examples/tts/conf/rad-tts_feature_pred.yaml

```diff
@@ -8,7 +8,7 @@ export_dir: ???
 sup_data_path: ???
 sup_data_types: ["log_mel", "align_prior_matrix", "pitch", "voiced_mask", "p_voiced", "energy"]
 
-whitelist_path: "nemo_text_processing/text_normalization/en/data/whitelist/tts.tsv"
+whitelist_path: null # default https://github.com/NVIDIA/NeMo-text-processing/blob/main/nemo_text_processing/text_normalization/en/data/whitelist/tts.tsv will be used
```

Comment:
seems to conflict with the above changes. Are you planning to host whitelist files in NeMo repo or the new NeMo-Text-Processing repo? why do we guide users to assign the full url instead of local files?

Reply:
discussed offline. removed this argument.